Consul/Operator.cs

```diff
@@ -421,6 +421,27 @@ public Task<QueryResult<AutopilotHealth>> AutopilotGetHealth(QueryOptions q, Can
         {
             return _client.Get<AutopilotHealth>("/v1/operator/autopilot/health", q).Execute(cancellationToken);
         }
+
+        /// <summary>
+        /// Retrieves the autopilot state of the cluster (synchronous version)
```

Comment:
I think that the "synchronous version" comment is misleading; this overload doesn't take query options, but it is not "synchronous" by any means. 
```suggestion
        /// Retrieves the autopilot state of the cluster
```